sql/synth_recipes.sql

```diff
@@ -4752,6 +4752,8 @@ INSERT INTO `synth_recipes` VALUES (75530,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,27
 INSERT INTO `synth_recipes` VALUES (75531,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4161,0,0,2781,2781,2781,2781,1,1,1,1,'Enfeeb. Kit Sleep');
 INSERT INTO `synth_recipes` VALUES (75532,1,0,0,0,0,0,34,0,0,0,4100,4242,12994,0,0,0,0,0,0,0,818,818,848,848,2,3,1,2,'Shoes (desynth)');
 INSERT INTO `synth_recipes` VALUES (75533,1,0,0,0,0,25,0,0,0,0,4100,4242,12722,0,0,0,0,0,0,0,834,819,819,819,2,5,6,7,'Bracers (desynth)'); -- FFXIclopedia
+INSERT INTO `synth_recipes` VALUES (75534,1,0,0,0,60,0,0,0,0,0,4100,4242,13445,0,0,0,0,0,0,0,745,745,745,745,1,2,2,2,'Gold_Ring (desynth)'); -- https://www.bg-wiki.com/ffxi/Gold_Ring
```

Comment:
1,2,2,2 would imply1 ingot when NQ, and 2 ingots on any HQ. That's not what tBG wiki sais

Reply:
ah yes you are correct, hq1 is also 1 ingot..